renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -170,7 +170,7 @@ struct IReplayOutput
   virtual ResourceId GetCustomShaderTexID() = 0;
   virtual bool PickPixel(ResourceId texID, bool customShader, uint32_t x, uint32_t y,
                          uint32_t sliceFace, uint32_t mip, uint32_t sample, PixelValue *val) = 0;
-  virtual uint32_t PickVertex(uint32_t eventID, uint32_t x, uint32_t y) = 0;
+  virtual uint32_t PickVertex(uint32_t eventID, uint32_t x, uint32_t y, uint32_t *pickedInstance) = 0;
```

Comment:
Okay - I can change it to be that way if you want.  I'm not 100% sure you can return structs to C# like that though, but another option would be to make it like PickPixel: return a bool and pass in a pointer to a PickedVertex.  That also makes the caller's code simpler because it doesn't need to know about the ~0U special value.